src/devices/README.md

```diff
@@ -21,6 +21,7 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
 ### Accelerometers
 
 * [ADXL345 - Accelerometer](Adxl345/README.md)
+* [ADXL357 - Accelerometer](Adxl357/README.md)
```

Comment:
Same here, no need. But you need to add a text file with the categories of the sensor.

Reply:
I removed it again.